### PR TITLE
Add API&Examples link to erlang docs

### PR DESCRIPTION
**Pull request by svrnm**
Contributes to https://github.com/open-telemetry/opentelemetry.io/issues/1808

https://deploy-preview-1943--opentelemetry.netlify.app/docs/instrumentation/erlang/api/
https://deploy-preview-1943--opentelemetry.netlify.app/docs/instrumentation/erlang/examples/

Screenshot 2022-10-28 at 10 40 50

### CODE REVIEW

File: content/en/docs/instrumentation/erlang/api.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: API reference
+linkTitle: API
+redirect: https://hexdocs.pm/opentelemetry_api/1.0.0-rc.2/OpenTelemetry.html
```
Comment (tsloughter) [inline content/en/docs/instrumentation/erlang/api.md:4]:
```suggestion
redirect: https://hexdocs.pm/opentelemetry_api/1.1.1/OpenTelemetry.html
```

**Review by tsloughter — APPROVED** [review]
Thanks! Updated the link to the API docs to the latest version, but other than that it looks good.

**Comment by svrnm** [comment]
> Thanks! Updated the link to the API docs to the latest version, but other than that it looks good.

is there a way to link to "latest" otherwise we need to update that on every release.

Also, I copied that link from opentelemetry-contrib-erlang repo README.md, maybe worth to check there as well

**Comment by svrnm** [comment]
@tsloughter it looks like `https://hexdocs.pm/opentelemetry_api/OpenTelemetry.html` is showing the latest version. Can I use that URL?

**Review by chalin — APPROVED** [review]

**Review by cartermp — APPROVED** [review]
LGTM, will merge in a day or so if there's no movement

**Comment by svrnm** [comment]
> > @tsloughter it looks like `https://hexdocs.pm/opentelemetry_api/OpenTelemetry.html` is showing the latest version. Can I use that URL?
> 
> Ah yes, good point, that is the best to use.

Done.

**Pull request merged**